scripts/apps/ingest/views/settings/ingest-sources-content.html

```diff
@@ -10,7 +11,7 @@
 
     <div class="sd-page__content">
         <ul class="pills-list provider-list">
-            <li ng-repeat="provider in providers._items" class="clearfix">
+            <li ng-repeat="provider in providers._items | filter:query" class="clearfix">
```

Comment:
There's pagination in the ingest-sources page, so it should search all pages.

Reply:
You don't need `| filter:query` anymore if you are filtering using the API. Do you need to change the markup at all in this case?

Reply:
> You don't need `| filter:query` anymore if you are filtering using the API. Do you need to change the markup at all in this case?

I didn't change any markup.
Design breaks are there because the branch was not updated with the master.